notebooks/NIRSpec_NSClean/FS_NSClean_example.ipynb

```diff
@@ -0,0 +1,1764 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #125.                        label="Cleaned (hand-modified mask)",
</pre><p>Hi, @kglidic. I wanted to suggest a small adjustment that may be beneficial. I noticed that the color scheme for the spectra is red, and using both red and green in the same figure may not be very accessible. To make the figure more inclusive, it might be helpful to add a new color, such as "purple" or any other color of your choice.</p>

<pre class="ql-syntax" spellcheck="false"> label="Cleaned (hand-modified mask)",
 color="Purple"
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Adopted this change